charger/daheimladen.go

```diff
@@ -0,0 +1,317 @@
+package charger
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/evcc-io/evcc/api"
+	"github.com/evcc-io/evcc/charger/daheimladen"
+	"github.com/evcc-io/evcc/util"
+	"github.com/evcc-io/evcc/util/request"
+)
+
+// DaheimLaden charger implementation
+type DaheimLaden struct {
+	*request.Helper
+	chargingStationID string
+	connectorID       int32
+	idTag             string
+	token             string
+	transactionID     int32
+}
+
+func init() {
+	registry.Add("daheimladen", NewDaheimLadenFromConfig)
+}
+
+// NewDaheimLadenFromConfig creates a DaheimLaden charger from generic config
+func NewDaheimLadenFromConfig(other map[string]interface{}) (api.Charger, error) {
+	cc := struct {
+		Token             string
+		ChargingStationID string
+		ConnectorID       int32
+		IDTag             string
+	}{}
```

Comment:
add `ConnectorID` default?

Reply:
Where does a user get `Token` and `ChargingStationID` from?

Reply:
The user will get the token from us, the chargingStationID is displayed on the charging station screen and the app itself

Reply:
Would it be ok to rename to `StationID`?